ci/build.yml

```diff
@@ -59,6 +59,16 @@ jobs:
   - task: PublishTestResults@2
     displayName: Publish test results
     condition: eq(variables['SPEECHSDK_RUN_TESTS'], 'true')
+  - bash: |
```

Comment:
Please either
- add `set -e` at the beginning to set that it stops on first failure (non-zero exit code) or
- chain the commands such as here
  ```
  a &&
  b &&
  c
  ```